spec/features/sdr_deposit_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe 'SDR deposit', type: :feature do
+  let(:start_url) { 'https://argo-stage.stanford.edu/' }
+  let(:api_url) { 'https://sdr-api-stage.stanford.edu' }
+  let(:source_id) { "testing:#{SecureRandom.uuid}" }
+  let(:apo) { 'druid:sc012gz0974' }
+  let(:collection) { 'druid:hv320hk4091' }
+  let(:catkey) { '10065784' }
+
+  before do
+    authenticate!(start_url: start_url, expected_text: 'Welcome to Argo!')
+  end
+
+  # This allows a login using credentials from the config gem.
+  class LoginFromEnv
+    def self.run
+      { email: ENV.fetch('SDR_EMAIL'), password: ENV.fetch('SDR_PASSWORD') }
+    end
+  end
+
+  it do
```

Comment:
It might be nice to have a little text description of the expectation here, but the test itself is also pretty easy to read.